client/gutenberg/editor/index.js

```diff
@@ -56,7 +56,7 @@ export default function () {
 		siteSelection,
 		redirect,
 		authenticate,
-		post,
+		redirectPostEditor,
```

Comment:
Since `redirectPostEditor` always redirects, do we need the `makeLayout` and `clientRender` middleware?

Reply:
Good catch, removed them and it still works properly.